coordinator/src/lib.rs

```diff
@@ -78,15 +77,6 @@ impl IntoResponse for AppError {
     }
 }
 
-/// Check if the liquidity is sufficient to open a JIT channel from the coordinator
-pub fn is_liquidity_sufficient(
```

Comment:
🤔 we might need something like that for our dlc channels? but on the other hand, if we reach that amount, an error is probably also fine.

Reply:
I removed it because we weren't using it!